extensions/liquibase-mongodb/runtime/src/main/java/io/quarkus/liquibase/mongodb/LiquibaseMongodbFactory.java

```diff
@@ -1,77 +1,101 @@
 package io.quarkus.liquibase.mongodb;
 
-import java.util.Map;
-import java.util.regex.Matcher;
-import java.util.regex.Pattern;
-
+import com.mongodb.client.MongoClient;
 import io.quarkus.liquibase.mongodb.runtime.LiquibaseMongodbBuildTimeConfig;
 import io.quarkus.liquibase.mongodb.runtime.LiquibaseMongodbConfig;
+import io.quarkus.mongodb.runtime.MongoClientBeanUtil;
 import io.quarkus.mongodb.runtime.MongoClientConfig;
 import liquibase.Contexts;
 import liquibase.LabelExpression;
 import liquibase.Liquibase;
 import liquibase.database.Database;
-import liquibase.database.DatabaseFactory;
+import liquibase.exception.DatabaseException;
+import liquibase.ext.mongodb.database.MongoConnection;
+import liquibase.ext.mongodb.database.MongoLiquibaseDatabase;
 import liquibase.resource.ClassLoaderResourceAccessor;
 
+import java.util.Map;
```

Comment:
I'm pretty sure moving the imports will break the formatting rules.

You can reformat the sources with `mvn process-sources`.

Also see https://github.com/quarkusio/quarkus/blob/main/CONTRIBUTING.md#ide-config-and-code-style